src/ngraph/runtime/executable.cpp

```diff
@@ -118,3 +118,8 @@ vector<runtime::PerformanceCounter> runtime::Executable::get_performance_data()
 {
     return vector<PerformanceCounter>();
 }
+
+bool runtime::Executable::save(std::ostream& output_stream)
+{
+    return false;
```

Comment:
Consider adding some error message or make it a pure virtual.

Reply:
I don't want to require that backends implement save/load. The error message should be handled by the caller. This call correctly returns a false indicating a failed save.